components/movementsensor/gpsrtk/gpsrtk.go

```diff
@@ -1,6 +1,6 @@
 // Package gpsrtk defines a gps and an rtk correction source
 // which sends rtcm data to a child gps
-// Experimental package
+// This is an Experimental package
```

Comment:
i missed this...why did we decide that rtk isn't beta-ready?

Reply:
I want to keep the door open for some config changes.

We have an 'rtk_station' concept that doesn't show anything and just sends data to child gps. Some of the station's functionality is repeated in the regular 'gps-rtk' that connects a specific chip and uses the internet to get correction data from there. There's probably a better and less user-heavy way of configuring these two models.